### DISCUSSION TITLE
I want to autoPlay video using MuxPlayer react

### DISCUSSION THREAD

**Discussion by murtaza-ch** [muxinc/elements#648] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Mux Elements/Packages does this apply to? Select all that apply

mux-player-react

### Description

I want to autoPlay video using MuxPlayer react.

### Expected Behavior

Currently, I'm not able to autoplay video without muting using mux player react. I know it is not recommended to autoplay without muting. but according to my business model, I need to autoplay the video without muting it like youtube playing their video.

I'm looking for a suitable solution for my use case. Please also provide any code snippets if you have any.

Thank you.

**Comment by gkatsev** [comment, marked as answer]
Autoplay nowadays is [pretty tricky](https://www.mux.com/blog/video-autoplay-considered-harmful) and may not be allowed to happen unless muted. Though, it depends on the browser settings for a particular user.
In Mux Player, [we have extended the regular autoplay property](https://docs.mux.com/guides/player/customize-look-and-feel#autoplay) to allow for easier configuration of autoplay. For example, with `autoplay="any"`, it'll try to autoplay unmuted and if the player cannot autoplay, it'll mute the player and try again.

Unfortunately, there is no way to force autoplay with sound.

**Comment by murtaza-ch** [comment]
Thank you for guiding me

**Comment by murtaza-ch** [comment]
HI, one more issue I'm facing that is PIP is not only playing on the current video page and it stops playing on other screens. can you please guide me on how to fix that issue?

**Reply by luwes** [reply]
could you explain a bit more? on my end the video stays playing in PiP when I visit a new page 
for example https://stream.new/v/fn7wPG6b3kSygzUw4GNXLmauDcKIT1G02

**Reply by murtaza-ch** [reply]
In your example, https://stream.new/v/fn7wPG6b3kSygzUw4GNXLmauDcKIT1G02 video in PIP mode also stopped when I visit a new page. for when I visit the Terms page. All I want is to keep the video playing in PIP mode when I visit any new page

**Reply by luwes** [reply]
hmm which OS / device / browser?

**Reply by murtaza-ch** [reply]
It's causing issues on all browsers. I tested it on Chrome and Safari as well. I'm using Macbook Pro M1 macOS Montery. 
<img width="1440" alt="Screenshot 2023-04-07 at 11 56 01 AM" src="https://user-images.githubusercontent.com/53378588/230558582-c64c60d5-ad52-491c-b295-1f3ed54be919.png">
<img width="1440" alt="Screenshot 2023-04-07 at 11 56 21 AM" src="https://user-images.githubusercontent.com/53378588/230558609-30535420-0ce6-4ab8-be15-075f407fdb93.png">
